.github/workflows/main.yml

```diff
@@ -0,0 +1,30 @@
+name: build
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: checkout-wrf_io
+      uses: actions/checkout@v2
+      with: 
+        path: wrf_io
+
+    - name: Installs
+      run: |
+        sudo apt-get install doxygen graphviz libnetcdf-dev libnetcdff-dev
+
+    - name: build_wrf_io
+      env: 
+        FC: gfortran-9
+        CC: gcc-9
+      run: |
+        set -x
+        sudo find / -name netcdf.h -print
```

Comment:
what is the purpose of this line?  Its a debug statement.

Reply:
Removed.